LICENSE.md

```diff
@@ -1,31 +1,21 @@
-As a work of the United States government, this project is in the
-public domain within the United States.
+# License
 
-Additionally, we waive copyright and related rights in the work
-worldwide through the CC0 1.0 Universal public domain dedication.
+As a work of the [United States of America's Government](https://www.usa.gov/), this project is in the public domain within the United States of America.
```

Comment:
`United States of America's Government` isn't a commonly used phrase. I'd prefer this to use common language, while of course remaining clear.

Reply:
Made the change!